samples/matter/common/src/app/matter_init.cpp

```diff
@@ -229,6 +233,11 @@ void DoInitChipServer(intptr_t /* unused */)
 	/* Set External Flash into sleep mode */
 	ExternalFlashManager::GetInstance().DoAction(ExternalFlashManager::Action::SLEEP);
 
+	/* Disable unused RAM blocks to reduce power consumption */
+#ifdef CONFIG_RAM_POWER_DOWN_LIBRARY
+	power_down_unused_ram();
```

Comment:
@ArekBalysNordic Sorry, for late review. What is the reason on calling this function? It's automatically run by the Power Down library, right?

Reply:
No, it is not called automatically anywhere. The documentation says about it as well: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/others/ram_pwrdn.html We need to call this function. We can consider modifying the RAM power-down library to force doing that at system boot once that kconfig is set.